packages/babel-traverse/src/index.js

```diff
@@ -81,6 +82,8 @@ traverse.clearNode = function (node) {
     if (key[0] === "_" && node[key] != null) node[key] = undefined;
   }
 
+  traverse.cache.path.delete(node);
```

Comment:
I'm struggling to see a case where this is something we need to clear in `clearNode`. If the node is moved, won't it just fail to be read from the cache later on?